snow/engine/snowman/block/batched_vm.go

```diff
@@ -33,6 +36,7 @@ type BatchedChainVM interface {
 
 func GetAncestors(
 	ctx context.Context,
+	log logging.Logger,
```

Comment:
Since we added a logger to this function should we log on the `database.ErrNotFound` lines as well now instead of silent return/break?

Reply:
I feel like `database.ErrNotFound` is expected behavior... so I don't really think it's needed... But if we wanted we could put a `Verbo` log here.